/**:
  ros__parameters:
    cruise_goals: ["11.91,-4.97","8.76,-3.12","8.78,1.19","12.75,2.38","11.91,-4.97","66.66, 66.66", "77.77, 77.77", "88.88, 88.88", "99.99, 99.99"]
    global_position_topic: "/global_costmap/published_footprint"
    cmd_vel_topic: "/cmd_vel"
    cmd_vel_remap_topic: "/cmd_vel_remap"
